components/console/helpers/table.rst

```diff
@@ -432,3 +432,24 @@ This will display the following table in the terminal:
     | Love    |
     | Symfony |
     +---------+
+
+.. tip::
+
+    You can create multiple lines using the :method:`Symfony\\Component\\Console\\Helper\\Table::addRows` method::
+
+        // ...
+        $table->addRows([
+            ['Hello', 'World'],
+            ['Love', 'Symfony'],
+        ]);
+        $table->render();
+        // ...
+
+    This generates:
```

Comment:
```suggestion
    This will output:
```
```suggestion
    This will display:
```

wdyt?

Reply:
I will send the correction with your second proposal.
Thank you.